### ISSUE TITLE
aCardWasUpdated event offer_event_rejected

### ISSUE THREAD

**Issue by oklotz** [sorare/api#318]
Hello,

I would like know when an offer is rejected. In aCardWasUpdated , we have event on accepted, cancelled but not for rejected. How to do, to know that?

Many Thanks

**Comment by J4F8** [comment]
yes, it would be a useful enhancement to get events when offers are rejected. Any update on this issue ?

**Comment by oklotz** [comment]
unfortunatly not
